modules/engine/src/test/scala/observe/engine/StepSpec.scala

```diff
@@ -185,7 +179,7 @@ class StepSpec extends CatsSuite {
     case _                       => false
   }
 
-  def runToCompletion(s0: TestState): Option[TestState] =
+  def runToCompletion(s0: TestState): IO[Option[TestState]] =
```

Comment:
This name may not be appropriate anymore as we are not really running the IO as before